<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormTestZhenDong</class>
 <widget class="QWidget" name="FormTestZhenDong">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FormTestZhenDong</string>
  </property>
  <widget class="GraphZhenDong" name="graph" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1004</width>
     <height>630</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>650</y>
     <width>600</width>
     <height>110</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>10</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>当前斗号：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="dounum">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>斗重量：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="weight_dou">
      <property name="styleSheet">
       <string notr="true">color: rgb(0,255,0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QImageLineEdit" name="val_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="whatsThis">
       <string>振动系数：?</string>
      </property>
      <property name="text">
       <string>振动系数：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>振动重量：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="weight_zd">
      <property name="styleSheet">
       <string notr="true">color: blue;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>650</y>
     <width>130</width>
     <height>110</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_start">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>650</y>
     <width>124</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_stop">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>709</y>
     <width>124</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cpt">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>650</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>计算</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_zero">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>710</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>置零</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QImageLineEdit</class>
   <extends>QLineEdit</extends>
   <header>MWidget.h</header>
  </customwidget>
  <customwidget>
   <class>GraphZhenDong</class>
   <extends>QWidget</extends>
   <header>GraphZhenDong.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>btn_back</tabstop>
  <tabstop>val_1</tabstop>
  <tabstop>btn_start</tabstop>
  <tabstop>btn_stop</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
